### PR TITLE
IGNITE-22467 Add getAssignments method to the PlacementDriver

### PR DESCRIPTION
**Pull request by sanpwc**
https://issues.apache.org/jira/browse/IGNITE-22467

### CODE REVIEW

File: modules/affinity/src/main/java/org/apache/ignite/internal/affinity/TokenizedAssignments.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+package org.apache.ignite.internal.affinity;
+
+import java.io.Serializable;
+import java.util.Set;
+
+/**
+ * Set of nodes along with associated token that is guaranteed to be changed if the set was changed.
+ */
+public interface TokenizedAssignments extends Serializable {
```
[the remark this answers is not in the document]
Comment (vldpyatkov) [reply modules/affinity/src/main/java/org/apache/ignite/internal/affinity/TokenizedAssignments.java:26]: In general, I approver the patch, these comments just a wishes.

**Comment by vldpyatkov** [comment]
In general, I approver the patch, these comments just a wishes.

**Review by vldpyatkov — APPROVED** [review]

File: modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java [hunk 2]
```diff
@@ -241,7 +252,49 @@ boolean isActiveActor() {
 
     /** Returns placement driver service. */
     public PlacementDriver placementDriver() {
-        return leaseTracker;
+        return new PlacementDriver() {
```
[the remark this answers is not in the document]
Comment (sanpwc) [reply modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java:255]: Fixed.

File: modules/affinity/src/main/java/org/apache/ignite/internal/affinity/TokenizedAssignmentsImpl.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+/**
+ * Set of nodes along with associated token that is guaranteed to be changed if the set was changed.
+ */
+public class TokenizedAssignmentsImpl implements TokenizedAssignments {
+    private static final long serialVersionUID = -6960630542063056327L;
+
+    @IgniteToStringInclude
+    private final Set<Assignment> nodes;
+
+    @IgniteToStringInclude
```
[the remark this answers is not in the document]
Comment (sanpwc) [reply modules/affinity/src/main/java/org/apache/ignite/internal/affinity/TokenizedAssignmentsImpl.java:35]: Fixed.

File: modules/affinity/src/main/java/org/apache/ignite/internal/affinity/TokenizedAssignments.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanpwc) [reply modules/affinity/src/main/java/org/apache/ignite/internal/affinity/TokenizedAssignments.java:26]: I'm not sure that we won't move impl to the common Assignments. So I'd rather leave the interface.

File: modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (sanpwc) [reply modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/PlacementDriverManager.java:273]: Fixed.

**Pull request merged**